### PR TITLE
JBPM-7692 Add edit feature on ExternalDataObjectsPresenter

### PR DESCRIPTION
**Pull request by bxf12315**
@cristianonicolai Please view it.

### CODE REVIEW

File: kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/settings/sections/externaldataobjects/ExternalDataObjectsItemView.html [hunk 1]
```diff
@@ -19,6 +19,9 @@
     <tr class="kie-tablelist__row" data-field="root">
         <td class="kie-tablelist__cell"><span data-field="type-name"></span></td>
         <td class="kie-tablelist__cell--actions">
+            <a class="btn btn-link" data-field="edit-button">
+                <span class="fa fa-pencil-square-o"></span>
+            </a>
             <a href="#0" data-field="remove-button">
```
Comment (cristianonicolai) [inline kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/settings/sections/externaldataobjects/ExternalDataObjectsItemView.html]: Please remove href

**Comment by cristianonicolai** [comment]
Depends on https://github.com/kiegroup/kie-wb-common/pull/2052

File: kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/settings/sections/externaldataobjects/ExternalDataObjectsItemPresenter.java [hunk 2]
```diff
@@ -36,8 +38,10 @@
     }
 
     @Inject
-    public ExternalDataObjectsItemPresenter(final View view) {
+    public ExternalDataObjectsItemPresenter(final View view,
+                                            AddImportPopup importPopup) {
```
Comment (cristianonicolai) [inline kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/settings/sections/externaldataobjects/ExternalDataObjectsItemPresenter.java]: Add final keyword to keep consistency?

File: kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/settings/sections/externaldataobjects/ExternalDataObjectsItemView.html (hunk 1 above)
Comment (bxf12315) [reply kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/settings/sections/externaldataobjects/ExternalDataObjectsItemView.html]: ok

File: kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/settings/sections/externaldataobjects/ExternalDataObjectsItemPresenter.java (hunk 2 above)
Comment (bxf12315) [reply kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-client/src/main/java/org/kie/workbench/common/screens/library/client/settings/sections/externaldataobjects/ExternalDataObjectsItemPresenter.java]: ok

**Comment by bxf12315** [comment]
@cristianonicolai updated

**Comment by bxf12315** [comment]
@cristianonicolai  old solution, so I close it.

**Pull request closed without merging**